### ISSUE TITLE
iOs app crashes on start after migration 1.3.61 -> 1.3.70

### ISSUE THREAD

**Issue by subprogram** [JetBrains/kotlin-native#3955]
```
dyld: Library not loaded: /System/Library/Frameworks/IOSurface.framework/IOSurface
  Referenced from: /Users/subprogram/Library/Developer/CoreSimulator/Devices/734560C9-6DF2-489F-839D-BE683FC7AD43/data/Containers/Bundle/Application/B95CB35D-07D0-4893-8A87-86837635D473/GuitarSongs.app/GuitarSongs
  Reason: no suitable image found.  Did find:
	/System/Library/Frameworks/IOSurface.framework/IOSurface: mach-o, but not built for iOS simulator
```
This issue is dependent on iOs version. In my tests only iOs 10.3.1 is affected.
I've also tested 12.4.4 and 13.1 - the issue is not reproduced.

**Comment by artdfel** [comment]
Hello, @subprogram! Sorry for such a late response. This problem seems to be caused by Kotlin/Native compiler caches, that was introduced in [1.3.70](https://blog.jetbrains.com/kotlin/2020/03/kotlin-1-3-70-released/#kotlin-native). Therefore, it can be workaround by disabling them.
Just set the `kotlin.native.cacheKind = none` in your project's Gradle properties.
Also, as the caches feature applies only for the debug mode, your releases should not be affected.

**Comment by subprogram** [comment]
Hi @artdfel! Thanks for your reply. I confirm that setting the `kotlin.native.cacheKind = none` solves the issue.

**Comment by udoy-touhid** [comment]
@artdfel. thanks for the heart. this single line of code saved my day, after I wasted more than 10 hours on this issues.

**Comment by SvyatoslavScherbina** [comment]
The fix for this issue will be shipped with 1.4.20.
So after updating your project to 1.4.20 please try to remove the "`kotlin.native.cacheKind = none`" workaround.
